### PR TITLE
example: add simplemind

### PR DESCRIPTION
**Pull request by mscolnick**
```
uvx marimo edit https://raw.githubusercontent.com/marimo-team/marimo/ms/simplemind-example/examples/ai/chat/simplemind_example.py
```

### CODE REVIEW

**Comment by kennethreitz** [comment]
awesome!

File: examples/ai/chat/simplemind_example.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
+# /// script
+# requires-python = ">=3.12"
+# dependencies = [
+#     "marimo",
+#     "simplemind==0.1.3",
```
Comment (kennethreitz) [inline examples/ai/chat/simplemind_example.py:5]: good job with the version pinning, as we're pre 1.0

Comment (mscolnick) [reply examples/ai/chat/simplemind_example.py:5]: marimo does this automatically (for reproducibility!). If you make breaks, we can (or you can, if you'd like) update the example

**Comment by kennethreitz** [comment]
where can i view this online?

**Comment by mscolnick** [comment]
> where can i view this online?

@kennethreitz we don't host them online, but instructions to run them locally are here: https://github.com/marimo-team/marimo/tree/main/examples/ai/chat

**Comment by kennethreitz** [comment]
I see! Thank you so much.

**Pull request merged**